pkg/kube/client.go

```diff
@@ -105,6 +109,50 @@ func (c *Client) Create(resources ResourceList) (*Result, error) {
 	return &Result{Created: resources}, nil
 }
 
+func (c *Client) Get(rel *release.Release) (string, error) {
```

Comment:
@bacongobbler 

Hi,
    I had modified the code as per your suggestion. I had gone through "kubectl get" implementation in depper. I found that Transformrequests is used to get the status of resources instead of rely on packages internal to kubernetes/kubernetes.

I had modified the code and tested, it worked fine. Could you please review it. You can check the resources output in below conservations.